app/upw/controllers/common.utils.js

```diff
@@ -38,26 +38,53 @@ const formatFlag = flag => flag.description || null
 
 const getRegistrations = async (crn, user) => {
   try {
-    const [, response] = await getRegistrationsForCrn(crn, user?.token, user?.id)
+    const { response, status } = await getRegistrationsForCrn(crn, user)
```

Comment:
Just checking - doesn't getRegistrationsForCrn return as an array - so [response, status] ?

Reply:
Ignore me - just saw the updates below :)